feat: Adds kraken GCP rights (#297)
meshcloud/meshcloud-docs: docs/meshstack.gcp.metering.md

roles/bigquery.dataViewer
```

> If you want to use the billing module of meshStack you also need another service principal, which is able to discover and list projects. We recommend to also create its own IAM role and assign it to the service account together with the following permissions:

Reviewer: ```suggestion
To enable the billing module of meshStack a service principal, which is able to discover and list projects, needs to be defined. We recommend to also create its own IAM role and assign it to the service account together with the following permissions:
```
